fdbserver/CommitProxyServer.actor.cpp

```diff
@@ -819,9 +819,15 @@ std::set<Tag> CommitBatchContext::getWrittenTagsPreResolution() {
 		// empty sets are sent to all logs.
 		return transactionTags;
 	}
+	if (pProxyCommitData->idempotencyClears.size()) {
+		return std::set<Tag>();
```

Comment:
maybe add a comment for this function that returning empty set means the batch will be sent to all tlogs.